deepchem/models/jax_models/layers.py

```diff
@@ -0,0 +1,100 @@
+import logging
+try:
+  import jax.numpy as jnp
+  import haiku as hk
+except:
+  raise ImportError('These classes require Jax and Haiku to be installed.')
+
+logger = logging.getLogger(__name__)
+
+
+class Linear(hk.Module):
+  """Protein folding specific Linear Module.
+
+  This differs from the standard Haiku Linear in a few ways:
+    * It supports inputs of arbitrary rank
+    * Initializers are specified by strings
+
+  This code is adapted from DeepMind's AlphaFold code release
+  (https://github.com/deepmind/alphafold).
+
+  Examples
+  --------
+  >>> import deepchem as dc
+  >>> import haiku as hk
```

Comment:
Currently, we do not support doc test support for Jax-dependencies.

Reply:
I've made a tweak to add jax to the `env.common.yml` to hopefully add doctest support. Is there any reason that wouldn't work for us?

Reply:
https://github.com/deepchem/deepchem/pull/2604/commits/86d025b2336749c8e66e5c452b4ff924916b5909

could you include it like this

Reply:
Done!